R/functions_util.R

```diff
@@ -58,28 +59,70 @@ scrnaseq_session_info = function(path_to_git=".") {
   return(out)
 }
 
+#' Returns a subsample of cells.
+#' 
+#' @param sc A Seurat sc object.
+#' @param num Number of cells to subsample.
+#' @param seed Seed for sampling. Default is 1.
+#' @param group Metadata colum group to consider for sampling with group_proportional.
+#' @param group_proportional Should the number of cells sampled from each group be proportional (TRUE) or should the number of cells be the same for each group (FALSE)? Only works if group is not NULL.
```

Comment:
My idea was that especially for heatmaps we sample similar numbers of cells per group so that visualisation is better. If you do `group_proportional` sampling, then the big cluster take up too much of the plot. So it you set `group_proportional` `FALSE`, it will try to sample similar number of cells for each group.